lib/libc/Kconfig

```diff
@@ -11,6 +11,14 @@ config REQUIRES_FULL_LIBC
 	  Helper symbol to indicate some feature requires a C library implementation
 	  with more functionality than what MINIMAL_LIBC provides
 
+config SUPPORT_FULL_LIBC
+	bool
+	depends on NATIVE_APPLICATION || TOOLCHAIN_HAS_NEWLIB
```

Comment:
I thought you needed `default y` for this to work. I think this change is turning off all those tests.

Reply:
oops! I think this can be 'def_bool y' instead of just 'bool'. Will try.

Reply:
(and, TOOLCHAIN_HAS_NEWLIB isn't a Kconfig variable anyways, so that doesn't work).